### PR TITLE
#170578497 - Implement Notifications Feedback

### PR DESCRIPTION
**Pull request by nignanthomas**
#### What does this PR do?
Implement Notifications Feedback.

#### Description of Task to be completed?
- Redirect to RegisterSuccess on successful registration
- Remove toast errors and used <Alert />
- Fix timestamp error
- Implement pagination in ViewRequests

#### How should this be manually tested?
- Clone ch-feedback-notifications-170578497 branch
- run `npm i`
- run `npm run dev`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#170578497](https://www.pivotaltracker.com/story/show/170578497)

#### Screenshots (if appropriate)
Screen Shot 2020-01-08 at 14 39 32
Screen Shot 2020-01-08 at 14 40 34
Screen Shot 2020-01-10 at 10 31 47

#### Questions:
N/A

### CODE REVIEW

**Review by ngireric123 — APPROVED** [review]
LGTM

**Review by sabin18 — APPROVED** [review]
Good job @nignanthomas

File: src/components/pages/requests/SearchBar.js [hunk 1]
```diff
@@ -81,8 +82,12 @@ export class SearchBar extends Component {
       this.setState((state) => ({ ...state, isLoading: true }));
       if (!checkManager()) {
         await props.managerSearchRequestAction(searchParams);
+        props.resetPageAction();
+        // props.cancelResetPageAction();
```
Comment (nakiwuge) [inline src/components/pages/requests/SearchBar.js]: remove commented code

File: src/components/pages/requests/SearchBar.js [hunk 2]
```diff
@@ -81,8 +82,12 @@ export class SearchBar extends Component {
...
       } else {
         await props.searchRequestAction(searchParams);
+        props.resetPageAction();
+        // props.cancelResetPageAction();
```
Comment (nakiwuge) [inline src/components/pages/requests/SearchBar.js]: remove commented code

**Pull request merged**